Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:22.15.0
+
+WORKDIR /app
+
+COPY . /app
+
+RUN npm install -g @angular/cli
+
+RUN npm install 
+
+EXPOSE 4200
+
+CMD ["ng", "serve", "--host", "0.0.0.0"]
```

You can directly do this as you already have script to run the project in `package.json`
```suggestion
CMD ["npm", "run", "start"]
```